frontend/src/static/js/v_authorship.js

```diff
@@ -6,26 +6,29 @@ const filesSortDict = {
   fileType: (file) => file.fileType,
 };
 
+function initialState() {
+  return {
+    isLoaded: false,
+    files: [],
+    selectedFiles: [],
+    filterType: 'checkboxes',
+    selectedFileTypes: [],
+    fileTypes: [],
+    filesLinesObj: {},
+    fileTypeBlankLinesObj: {},
+    filesSortType: 'lineOfCode',
+    toReverseSortFiles: true,
+    searchBarValue: '',
+  };
+}
+
```

Comment:
Should this be an object instead of a function?

Reply:
It should be a function as I need to reset the state of the component while it is still alive by creating a new object, as global objects persist.